crates/l2/prover/src/cli.rs

```diff
@@ -11,14 +11,19 @@ pub struct ProverCLI {
 #[derive(Parser)]
 pub struct ProverClientOptions {
     #[arg(
-        long = "prover-server-endpoint",
-        value_name = "PROVER_SERVER_ENDPOINT",
-        env = "PROVER_CLIENT_PROVER_SERVER_ENDPOINT",
-        help = "Endpoint address where the server is running",
-        help_heading = "Prover client options",
-        default_value = "localhost:3900"
+        long = "http.addr",
+        value_name = "IP_ADDRESS",
+        env = "PROVER_CLIENT_PROVER_CLIENT_ADDRESS",
+        help_heading = "Prover client options"
+    )]
+    pub http_addr: String,
+    #[arg(
+        long = "http.port",
+        value_name = "PORT",
+        env = "PROVER_CLIENT_PROVER_CLIENT_PORT",
+        help_heading = "Prover client options"
     )]
-    pub prover_server_endpoint: String,
+    pub http_port: u16,
```

Comment:
Not sure if this is correct. The old `prover_server_endpoint` is the URL to connect to the new `ProofCoordinator`. For this cases, I think asking for a URL is better.

Reply:
Ok, I tried to be consistent with `ethrex` CLI, but it's true that we currently use `Url` for `eth_rpc_url`.

Reply:
Oh, remembered another reason to do as `ethrex` does. We do not import the `reqwest` crate in the `prover` crate. I'd prefer to avoid adding that dependency for the sole reason of using `Url` here.